corehq/apps/app_manager/static/app_manager/template_apps/agriculture/multimedia.json

```diff
@@ -0,0 +1,28 @@
+{
+  "commcare/image/module0_en.png":
+          "https://cdn2.hubspot.net/hubfs/503070/images/appcues%20template%20app/aggriculture/module0_en.png",
```

Comment:
That's a good question for @orangejenny 

She made the change and started hosting them on HubSpot here: https://github.com/dimagi/commcare-hq/commit/e9fb3b5934d15fcbb1008ce0d29c480d8aecc579
> This is just to get these multimedia files out of the commcare-hq repo,
since they're large and there may be churn around the content of the template app.